web/api/v1/api.go

```diff
@@ -700,6 +700,16 @@ func (api *API) labelNames(r *http.Request) apiFuncResult {
 	if names == nil {
 		names = []string{}
 	}
+
+	// Respect the "limit" parameter.
```

Comment:
Probably also not worth adding all these comments because it's clear what this does.